src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -198,6 +206,60 @@ impl HummockVersionExt for HummockVersion {
             .map(|group| group.levels.len() + 1)
             .unwrap_or(0)
     }
+
+    fn calculate_compaction_group_statistic(&self) -> Vec<TableGroupInfo> {
+        let mut infos = vec![];
+        for (group_id, group) in &self.levels {
+            let group_size = group
+                .levels
+                .iter()
+                .map(|level| level.total_file_size)
+                .sum::<u64>()
+                + group.l0.as_ref().unwrap().total_file_size;
+            // only calculate the bottommost too level.
```

Comment:
The implementation does not match the comments, we are still counting all levels instead of the bottommost level? (sst.table_ids.len() == 1 may be satisfied when cutting by table)

Reply:
Yes. we could calculate all levels if the sstable file only inlcude one state-table